tests/web/conftest.py

```diff
@@ -8,7 +11,7 @@
 
 
 @pytest.fixture
-def project_tmp_path(tmp_path: Path) -> Path:
+def project_tmp_path(tmp_path: Path) -> t.Generator[Path, None, None]:
```

Comment:
this can just be like iterator right? or just remove the types

Reply:
I'll remove the types.